### PR TITLE
TP-Link Archer MR200-v4 support added

### PR DESCRIPTION
**Pull request by MiKuBB**
This add support for TP-Link MR200-v4

### CODE REVIEW

File: target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds [hunk 1]
```diff
@@ -82,6 +82,11 @@ tplink,archer-c50-v4)
 	ucidef_set_led_wlan "wlan2g" "wlan2g" "green:wlan2g" "phy0tpt"
 	ucidef_set_led_wlan "wlan5g" "wlan5g" "green:wlan5g" "phy1tpt"
 	;;
+tplink,mr200v4)
+	ucidef_set_led_netdev "lan" "lan" "white:lan" "eth0.1"
+	ucidef_set_led_netdev "wan" "wan" "white:wan" "eth0.2"
+	set_wifi_led "white:wlan"
+;;
```
Comment (obsy) [inline target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds:89]: no indentation

File: target/linux/ramips/image/mt76x8.mk [hunk 2]
```diff
@@ -598,7 +598,7 @@ define Device/tplink_mr200-v4
   $(Device/tplink-v2)
   SUPPORTED_DEVICES := tplink,mr200-v4
   IMAGE_SIZE := 7872k
-  DEVICE_TITLE := TP-Link ArcherMR200 v4
+  DEVICE_TITLE := TP-Link Archer MR200 v4
```
Comment (obsy) [inline target/linux/ramips/image/mt76x8.mk:601]: Squash this into one commit

**Review by obsy — CHANGES_REQUESTED** [review]

File: target/linux/ramips/mt76x8/base-files/etc/board.d/02_network [hunk 3]
```diff
@@ -284,6 +284,11 @@ ramips_setup_macs()
 		;;
 	tplink,archer-c20-v4|\
 	tplink,archer-c50-v3|\
+	tplink,mr200-v4)
+		wan_mac=$(macaddr_add "$(mtd_get_mac_binary rom 0xf100)" 1)
+                ucidef_add_switch "switch0" \
+                        "1:lan" "2:lan" "3:lan" "4:wan" "6t@eth0"
```
Comment (musashino205) [inline target/linux/ramips/mt76x8/base-files/etc/board.d/02_network:290]: This should be moved to ```ramips_setup_interfaces()```

**Review by MiKuBB — COMMENTED** [review]
thx

**Comment by andyboeh** [comment]
You are missing a proper commit message and all commits should be squashed into one - amongst others. Please see here for the guidelines: https://openwrt.org/submitting-patches

**Comment by MiKuBB** [comment]
> You are missing a proper commit message and all commits should be squashed into one - amongst others. Please see here for the guidelines: https://openwrt.org/submitting-patches

**Comment by MiKuBB** [comment]
iam lost

**Pull request closed without merging**